app/uk/gov/hmrc/transitmovementsvalidator/services/XmlValidation.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsvalidator.services
+
+import uk.gov.hmrc.transitmovementsvalidator.models.MessageTypeXml
+
+import javax.xml.XMLConstants
+import javax.xml.parsers.SAXParserFactory
+import javax.xml.validation.SchemaFactory
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+trait XmlValidation {
```

Comment:
Do we want these traits, or do we perhaps want to consider having two separate validation services altogether, XML and Json (or do we just want the one class and not bother?)